crates/notion-core/src/env.rs

```diff
@@ -20,3 +20,14 @@ pub fn path_for(version: &str) -> OsString {
     path_vec.extend(split);
     env::join_paths(path_vec.iter()).unwrap()
 }
+
+/// Produces a modified version of the current `PATH` environment variable that
+/// removes the Notion shims and binaries.
+pub fn path_no_notion() -> OsString {
```

Comment:
Nit: I think the name `path_without_notion` reads better than `path_no_notion`.

Reply:
Actually, `path_for` and `path_no_notion` are highly similar but the differences are subtle and non-obvious, and the names don't make the differences clear. It would be better to clarify in both the names and the comments why `path_for` is the right thing for invoking Node, and `path_no_notion` is the right thing for invoking 3p binaries.

Reply:
I'm changing these to `path_for_installed_node` and `path_for_system_node`, which is a little clearer about when these should be used